about.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/2b1476549e.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="assets/stylesheets/mobile.css" media="screen and (max-width: 768px)">
+  <link rel="stylesheet" href="assets/stylesheets/desktop.css" media="screen and (min-width: 768px)">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">             
+  <title> About Page</title>
```

Comment:
- [ ] [OPTIONAL]: You should consider changing this title to something more meaningful, for instance, "Web Development Course - About Page".